src/test/java/org/jenkinsci/plugins/saml/SamlFormValidationsTest.java

```diff
@@ -73,132 +73,132 @@ public void start() {
     @LocalData("testReadSimpleConfigurationAdvancedConfiguration")
     @Test
     public void testIdpMetadata() throws Exception {
-        assertEquals(idpMCdescriptor.doTestIdpMetadata(null).kind, Kind.ERROR);
-        assertEquals(idpMCdescriptor.doTestIdpMetadata("").kind, Kind.ERROR);
-        assertEquals(idpMCdescriptor.doTestIdpMetadata(" ").kind, Kind.ERROR);
+        assertEquals(Kind.ERROR, idpMCdescriptor.doTestIdpMetadata(null).kind );
```

Comment:
I guess that inverts all the assertEquals is for a better read of the asserts but it is not related to the change, it uses to be in another PR, but it is fine.